### PR TITLE
:construction_worker: Add travis-ci with mypy and black

### LINKED ISSUE #11: travis-ci with black and mypy

### PR DESCRIPTION
**Pull request by b0g3r**
closes #11

### CODE REVIEW

**Comment by b0g3r** [comment]
Related to #2 

Because extractor functions get an answer as `Dict[str, Any]` mypy doesn't know what expects for each dict member: 
https://github.com/piterpy-meetup/c4p2n/blob/77b962a53796c270ba121c518f328b4067c17289/c4p2n/extractors.py#L4-L17

File: c4p2n/extractors.py [hunk 1]
```diff
@@ -2,25 +2,29 @@
...
 def extract_answer(answer: Dict[str, Any]) -> Union[str, List[str]]:
     answer_type = get_answer_type(answer)
-    extractor = answer_type_to_extractor[answer_type]
-    return extractor(answer)
+    if answer_type == "text":
+        return extract_text(answer)
+    elif answer_type == "choices":
+        return extract_choices(answer)
+    else:
+        raise TypeError(f"Unexpected answer type: {answer_type}")
```
Comment (ave2me) [inline c4p2n/extractors.py:30]: Do we really now need `answer_type_to_extractor` then? I myself would prefer to use a mapping-style here but this is also OK if it does its job and satisfies mypy checks.

**Review by ave2me — COMMENTED** [review]
Everything looks fine for me except for changes to `extract_answer` which are debatable. Want to see more on why the new style is preferable over using the mapping.

File: c4p2n/extractors.py (hunk 1 above)
Comment (ave2me) [reply c4p2n/extractors.py:30]: Well, actually, guess this is not that important.

**Pull request merged**